test/index.html

```diff
@@ -18,6 +18,10 @@
     background: yellow;
   }
 
+  .u-hidden {
```

Comment:
Ah that was just to hide the svg symbol reference so it didn't have to be copied over and over.